includes/html/api_functions.inc.php

```diff
@@ -2601,9 +2601,11 @@ function list_logs(Illuminate\Http\Request $request, Router $router)
         $param[] = $to;
     }
 
+    $sort_order = $request->get('sortorder') === 'DESC' ? 'DESC' : 'ASC';
+
     $count_query = $count_query . $query;
     $count = dbFetchCell($count_query, $param);
-    $full_query = $full_query . $query . " ORDER BY $timestamp ASC LIMIT $start,$limit";
+    $full_query = $full_query . $query . " ORDER BY $timestamp $sort_order LIMIT $start,$limit";
```

Comment:
this should be inserted via $param

Reply:
PDO doesn't allow binding outside of the prepared statement as far as I'm aware.  Which is why I assume you're interpolating timestamp, start and limit.